lightningd/chaintopology.c

```diff
@@ -656,7 +656,7 @@ static void json_dev_setfees(struct command *cmd,
 static const struct json_command dev_setfees_command = {
 	"dev-setfees",
 	json_dev_setfees,
-	"Set feerate in satoshi-per-kw for {immediate}, {normal} and {slow} (each optional) and show the value of those three feerates"
+	"Set feerate in satoshi-per-kw for {immediate}, {normal} and {slow} (each is optional, when set, separate by spaces) and show the value of those three feerates"
```

Comment:
> when set, separate by spaces

Hmm this is kind of implied since this is the way all lightning-cli calls work, so it's a bit superfluous imo

Reply:
It was less obvious for me. Debug shows values separated by / so I found myself trying out some variants before I got it right

Reply:
It's only separated by spaces for lightning-cli.

We really need to implement `lightning-cli help [cmd]`.  Will start a PR...